zend_abstract_interface/zai_sapi/zai_sapi.h

```diff
@@ -20,6 +20,10 @@
 ZEND_TSRMLS_CACHE_EXTERN()
 #endif
 
+#if PHP_VERSION_ID >= 80000
+#define TSRMLS_DC
+#endif
+
```

Comment:
Instead of exposing `TSRMLS_DC` via the header, would you be open to moving `register_custom_server_variables` down to the bottom where the PHP version-specific definitions are?